.github/workflows/build-ci-infra-images.yaml

```diff
@@ -7,6 +7,9 @@ on:
     paths-ignore:
     - 'docs/**'
     - '**/*.md'
+    - '*.md'
```

Comment:
Dumb question: isn't this covered by the above pattern?
At least that's what https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-file-paths says, IIUC.

Reply:
you are right. I was not sure whether a pattern could start with a `/ ` because no example starts with a slash, but the explanation of the`'**/README.md'` example in your link says so